go.mod

```diff
@@ -394,6 +395,7 @@ require (
 	knative.dev/pkg v0.0.0-20220325200448-1f7514acd0c2 // indirect
 	nhooyr.io/websocket v1.8.7 // indirect
 	oras.land/oras-go v0.4.0 // indirect
+	sigs.k8s.io/controller-tools v0.7.0
```

Comment:
I thought with more recent go, this section is for `indirect` deps only? Won't `go mod tidy` kick it out?

Reply:
Right, I moved it up